### PR TITLE
Compatibility with Conjur Enterprise

### PR DESCRIPTION
**Pull request by micahlee**
The purpose of this PR is to make the secret rotation example work also with Conjur Enterprise.

The overall architecture is the same, with a producer and consumer application. Each application has its own machine identity through Conjur, and there is a shared secret between them.

The differences between the Enterprise and Open Source demos are:
*  It uses CyberArk Conjur Enterprise Appliance container image rather than CyberArk Conjur Open Source
* It addresses TLS certificate validation for the appliance
* It uses the Conjur Enterprise V4 conventions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,9 +61,8 @@ To run this system demo, you will need:
 
 2. Load the Conjur policy
 
-    > NOTE: After the first run of the demo, you will be prompted to overwrite
-    > the local conjur-cli configuration. You should enter `yes` for this prompt.
-
+    > NOTE: You can review the policy [here](cli/policy.yaml).
```
Comment (ryanprior) [inline README.md:64]: I like this

File: enterprise/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+To run this system demo, you will need:
+
+1. Docker and Docker Compose
+
+2. Access to a Conjur Enterprise Appliance Docker container image
+
+## Running the Demo
+
+1. Navigate to the `enterprise/` directory in the depo repository
```
Comment (ryanprior) [inline enterprise/README.md:20]: One of the steps will have to be building the `helloworld` and `consumer` container images, if you haven't already.

**Review by ryanprior — CHANGES_REQUESTED** [review]
Super, just needs a small doc fix.

File: enterprise/bin/start-conjur [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+
+conjur_url=${CONJUR_URL:-https://conjur}
+conjur_account=${CONJUR_ACCOUNT:-cucumber}
+conjur_admin=${CONJUR_ADMIN_LOGIN:-admin}
+
+# Pull required container images from Docker Hub
+echo 'Pulling Docker images...'
+docker-compose pull --quiet conjur
+(cd .. && docker-compose build helloworld consumer)
```
Comment (micahlee) [inline enterprise/bin/start-conjur:10]: ⬆️ The images for helloworld and consumer are built here

**Pull request merged**